pkgs/by-name/op/open-timeline-io/package.nix

```diff
@@ -0,0 +1,5 @@
+{
+  type = "alias";
+  release = "25.11";
+  package = "opentimelineio";
+}
```

Comment:
Ah… so the reason for the mandatory `type = "alias";` is that we don’t have a wrapper function here at all.

But this is no good: you can have `fooFull` replaced with `foo.override { withStuff = true; }` and this does happen.

I know this ties in to supporting more non‐package stuff in `by-name`, but I think this would at least have to be `package = { opentimelineio }: opentimelineio;` or something. (Also, we should probably call it `target` or something rather than `package`, because we can have aliases for package sets too.)

Is the idea that this makes `.override` work properly, compared to `{ mkAlias, opentimelineio }: mkAlias { release = "25.11"; target = opentimelineio; }`? Or is there a more fundamental reason that won’t work with this approach?